sketches/language-basics/language-basics.go

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2016 Leandro Pereira <leandro@tia.mat.br>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+func assignOps() {
+	someVar := 6 * 7;
```

Comment:
maybe we could keep those as unit tests, instead of separate end to end conversion tests. I'd like to keep the sketches directory for actual Arduino "sketches"